### PR TITLE
(Re)Enable EIP IPv6 E2Es and fix issue with test

### PR DESCRIPTION
**Pull request by martinkennelly**
Issue was healthcheck test didnt consider IPv6 when adding iptable rule to drop packets therefore test would fail because failover didnt occur.

Fixes: #4130

### CODE REVIEW

**Comment by martinkennelly** [comment]
Save E2E flake https://github.com/ovn-org/ovn-kubernetes/issues/3986

**Comment by martinkennelly** [comment]
Rebased only

File: test/e2e/util.go [hunk 1]
```diff
@@ -1061,31 +1061,28 @@ func allowOrDropNodeInputTrafficOnPort(op, nodeName, protocol, port string) {
...
+		notFound1 := "No chain/target/match by that name"
+		notFound2 := "does a matching rule exist in that chain?"
+		notFound := err != nil && (strings.Contains(err.Error(), notFound1) || strings.Contains(err.Error(), notFound2))
+		if err != nil && !notFound {
+			framework.Failf("failed to check existance of %s rule on node %s: %v", iptables, nodeName, err)
+		}
+		if op == "delete" && notFound {
+			// rule is not there
+			return
+		} else if op == "append" && err == nil {
```
[the remark this answers is not in the document]
Comment (martinkennelly) [reply test/e2e/util.go:1078]: ya, it looks like it. I didnt want to make changes to code I didnt write here. It looks like no func error here.

[the remark this answers is not in the document]
Comment (jcaamano) [reply test/e2e/util.go:1078]: looks like a low hanging fruit though, who knows, might be related to one of our many flakes.

Have you tried replacing `append` with `insert` here to check what happens to the tests?

[the remark this answers is not in the document]
Comment (martinkennelly) [reply test/e2e/util.go]: works fine - changed it

**Review by jcaamano — APPROVED** [review]

**Pull request merged**